### PR TITLE
Add author tags to code, stream meetings for UniqueMeetingList

### PR DESCRIPTION
**Pull request by jeffreyooi**
This PR includes:
- Add author tags to indicate code authorship so that Reposense attribute code correctly
- Change `UniqueMeetingList` to stream and collect meetings from `UniqueGroupList` as per requested
- Bug fix:
   - Fix time on meeting card not showing the correct number of digits #203 
   - Fix UI not refreshing group card after meet / cancel command is called due to change not observed by UI. (Implemented workaround by raising event to request ListView to refresh)
   - Fix unable to undo meet / cancel command (#153)

### CODE REVIEW

**Comment by jeffreyooi** [comment]
@Derek-Hardy fixed equality test with null. Thanks for bringing this up!

File: src/main/java/seedu/meeting/model/meeting/UniqueMeetingList.java [hunk 1]
```diff
@@ -107,4 +58,14 @@ public boolean equals(Object other) {
                 || (other instanceof UniqueMeetingList // instanceof handles nulls
                 && internalList.equals(((UniqueMeetingList) other).internalList));
     }
+
+    /**
+     * Comparator for {@code Meeting} to sort meetings by time.
+     */
+    static class MeetingComparator implements Comparator<Meeting> {
```
Comment (NyxF4ll) [inline src/main/java/seedu/meeting/model/meeting/UniqueMeetingList.java]: I remember that meeting already have a built in comparator

Comment (jeffreyooi) [reply src/main/java/seedu/meeting/model/meeting/UniqueMeetingList.java]: Resolved in latest commit

Comment (NyxF4ll) [reply src/main/java/seedu/meeting/model/meeting/UniqueMeetingList.java]: ok

**Pull request merged**